src/style.css

```diff
@@ -2,0 +1,85 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  margin: 0;
+}
+
+.input {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  font-family: Avenir, Avenir-Heavy, Arial, Helvetica, sans-serif;
```

Comment:
Here is where you should specify your fallback fonts. However, ensure that the font names match the names specified in the `font-face` rules. For example, you have `Avenir-Heavy` as a separate font, so it should not be a fallback for `Avenir`. (CSS issue)